movement/monge_ampere.py

```diff
@@ -241,8 +257,19 @@ def l2_projector(self):
 
 class MongeAmpereMover_Relaxation(MongeAmpereMover_Base):
     r"""
-    The elliptic Monge-Ampere equation is solved in a parabolised form using a
-    pseudo-time relaxation,
+    The standard, elliptic form of the Monge-Ampère equation used for mesh movement takes
+    the form,
+
+    .. math::
+        m(\mathbf{x})\det(I + H(\phi)) = \theta,
+
+    for a convex scalar potential :math:`\phi`. Here :math:`I` is the identity matrix,
+    :math:`\theta` is a normalisation coefficient and :math:`H(\phi)` denotes the Hessian
+    of :math:`\phi` with respect to the coordinates :math:`\xi` of the computational
+    mesh.
```

Comment:
Yes, the MA equation is copied and pasted, but it's mainly for context. The quasi-Newton approach solves it directly whereas the relaxation approach solves it in parabolised form. Personally, I think it's fine to mention the equation being solved in each docstring.

I'd rather not put important information in the `MongeAmpereMover_Base` base class because it's unlikely that users would work with it. They're more likely to use the `MongeAmpereMover` factory function.

Reply:
Sounds good! :)

Reply:
I think the rest should be addressed in 39bca110170eea92443cdb3919cb45c6d9332531.